allensdk/brain_observatory/ecephys/ecephys_session.py

```diff
@@ -1036,7 +1039,7 @@ def get_stimulus_parameter_values(
 
             non_null = np.array(uniques[uniques != "null"])
             non_null = non_null
-            non_null = np.sort(non_null)
+            # non_null = np.sort(non_null)
```

Comment:
Did you leave this here for testing? If it's not going to be used you should remove the line.